radix-engine-tests/tests/account_deposit_modes.rs

```diff
@@ -463,16 +459,16 @@ impl AccountDepositModesTestRunner {
         deposit_method: DepositMethod,
         sign: bool,
     ) -> TransactionReceipt {
-        let (method, is_vec) = match deposit_method {
-            DepositMethod::Deposit => (ACCOUNT_DEPOSIT_IDENT, false),
-            DepositMethod::TryDeposit => (ACCOUNT_TRY_DEPOSIT_OR_REFUND_IDENT, false),
-            DepositMethod::TryDepositOrAbort => (ACCOUNT_TRY_DEPOSIT_OR_ABORT_IDENT, false),
-            DepositMethod::DepositBatch => (ACCOUNT_DEPOSIT_BATCH_IDENT, true),
+        let (method, is_vec, insert_badge) = match deposit_method {
```

Comment:
Minor: Why not just turn this match statement into a `DepositMethod` method so it can be reused.

Reply:
Good point, will change that.